services/galley/src/Galley/Schema/Run.hs

```diff
@@ -87,8 +87,8 @@ import Galley.Schema.V84_MLSSubconversation qualified as V84_MLSSubconversation
 import Galley.Schema.V85_MLSDraft17 qualified as V85_MLSDraft17
 import Galley.Schema.V86_TeamFeatureMlsMigration qualified as V86_TeamFeatureMlsMigration
 import Galley.Schema.V87_TeamFeatureSupportedProtocols qualified as V87_TeamFeatureSupportedProtocols
-import Galley.Schema.V88_TruncateMLSGroupMemberClient qualified as V88_TruncateMLSGroupMemberClient
-import Galley.Schema.V89_RemoveMemberClient qualified as V89_RemoveMemberClient
+import Galley.Schema.V88_RemoveMemberClient qualified as V89_RemoveMemberClient
+import Galley.Schema.V89_TruncateMLSGroupMemberClient qualified as V88_TruncateMLSGroupMemberClient
```

Comment:
In the PR description you say: "The migrations Remove member_client and Truncate mls_group_member_client both had the version 88". At least the module names had the numbers right. However, why swapping their numbers?

I see further down that both had `Migration 88`, but I am not sure what the consequence of that is/was.

Reply:
Both were run, but the last one overwrote the entry in the meta table.